Automattic/woocommerce-services
# Remove Stripe deferred account creation methods

As of May 31, the API for stripe deferred account activation will no
longer be supported. As a result, the methods and class related to this
functionality are no longer needed.

See paJDYF-x4-p2

This removes the `WC_REST_Connect_Stripe_Account_Controller` class along
with some related methods in `WC_Connect_API_Client` and
`WC_Connect_Stripe`.

**Testing Instructions**

- With the changes implemented in https://github.com/woocommerce/woocommerce-admin/pull/4164 checked out, run through the Stripe connection flow in onboarding.
- Confirm there are no errors and that Stripe settings are displayed correctly after finishing the above.
